phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/ViewsConfiguration.java

```diff
@@ -41,27 +43,46 @@
     @Value("${application.name}")
     private String applicationName;
 
+    @Value("${application.theme:defaultTheme.css}")
+    private String theme;
+
     @Autowired
     private ViewsConfigurationLibrary viewsConfigurationLibrary;
 
     @Bean
+    public ThemeManager themeManager() {
+        switch (theme) {
+            case "breezeDark":
+                return new ThemeManager(Themes.BREEZE_DARK);
+            case "dark":
```

Comment:
This logic should be implemented in Themes enum (which name should be singular btw)